homeassistant/components/tailwind/number.py

```diff
@@ -0,0 +1,86 @@
+"""Number entity platform for Tailwind."""
+from __future__ import annotations
+
+from collections.abc import Awaitable, Callable
+from dataclasses import dataclass
+from typing import Any
+
+from gotailwind import Tailwind, TailwindDeviceStatus
+
+from homeassistant.components.number import NumberEntity, NumberEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE, EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import TailwindDataUpdateCoordinator
+from .entity import TailwindEntity
+
+
+@dataclass(kw_only=True)
+class TailwindNumberEntityDescription(NumberEntityDescription):
+    """Class describing Tailwind number entities."""
+
+    value_fn: Callable[[TailwindDeviceStatus], int]
+    set_value_fn: Callable[[Tailwind, float], Awaitable[Any]]
+
+
+DESCRIPTIONS = [
+    TailwindNumberEntityDescription(
+        key="brightness",
+        icon="mdi:led-on",
+        translation_key="brightness",
+        entity_category=EntityCategory.CONFIG,
+        native_step=1,
+        native_min_value=0,
+        native_max_value=100,
+        native_unit_of_measurement=PERCENTAGE,
+        value_fn=lambda data: data.led_brightness,
+        set_value_fn=lambda tailwind, brightness: tailwind.status_led(
+            brightness=int(brightness),
+        ),
+    ),
+]
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Tailwind number based on a config entry."""
+    coordinator: TailwindDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities(
+        TailwindNumberEntity(
+            coordinator=coordinator,
+            description=description,
```

Comment:
```suggestion
            coordinator,
            description,
```

Reply:
Same as above.

Reply:
Applied in [93aec43](https://github.com/home-assistant/core/pull/105926/commits/93aec43f1181b93c79d868dcb5571242259be00d)